### DISCUSSION TITLE
[Docs] 官网右侧滚动条有点窄了，并且颜色太浅，不太好操作

### DISCUSSION THREAD

**Discussion by warmthsea** [element-plus/element-plus#8746] [Feature Request]
Feature type: **`Docs`**

## Details

- Enhanced component name: `All`

## Description

官网右侧滚动条有点窄了，并且颜色太浅，不太好操作

### Additional comments

_(empty)_

![image](https://user-images.githubusercontent.com/45450994/178173729-cbbec64b-17a2-4cda-b6af-335bd7616a9d.png)

**Comment by KimYangOfCat** [comment]
页面的滚动一般都是鼠标滚轮吧，有什么高发的场景需要去点击或拖拽这个滚动条吗？

element UI 就是认为游览器默认的滚动条太粗太丑，才改的，目前的样式我个人认为是OK的，也没有窄到点击误触率很高的地步。我个人日常使用基本不会去拖拽或点击这个滚动条，偶尔需要这样做的时候也没有发生误触或未响应的状态。

不过颜色有点儿浅倒是很赞同，不清楚为什么文档的滚动条的相比 Scrollbar 组件滚动条的默认颜色会浅一些，颜色太浅确实不容易看见，也许恢复默认的颜色就好了
![image](https://user-images.githubusercontent.com/32960305/179382332-8b9aadb9-6ff1-474c-a823-3b2d7fbaea16.png)

**Reply by warmthsea** [reply]
可能是我鼠标滚轮坏了，我习惯拉滚动条

**Reply by KimYangOfCat** [reply]
啊这，哈哈，那你应该考虑买个新鼠标了
